pkgs/servers/monitoring/vmctl/default.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+buildGoModule rec {
+  pname = "vmctl";
+  version = "1.93.1";
+
+  src = fetchFromGitHub {
+    owner = "VictoriaMetrics";
+    repo = "VictoriaMetrics";
+    rev = "v${version}";
+    sha256 = "sha256-52OY4sZ2UI+p6+QYkRW8Ov9SbLPKmvgvgV7rhfo0tAY=";
+  };
+
+  ldflags = ["-s" "-w" "-X github.com/VictoriaMetrics/VictoriaMetrics/lib/buildinfo.Version=${version}"];
+
+  vendorHash = null;
+
+  subPackages = ["app/vmctl"];
+
+  meta = with lib; {
+    homepage = "https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/app/vmctl";
+    description = "VictoriaMetrics command-line tool";
+    license = licenses.asl20;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [nullx76];
```

Comment:
@viktornordling  it seems like you have entered a different maintainer than you. Or is this an alternative account of yours? @NULLx76  is this okay with you to take maintainership of this package ?

Reply:
I wouldn't mind being co-maintainer, but I do think it's nice if @viktornordling also lists themselves as a maintainer.

Reply:
Yeah, sorry, I assumed the maintainer listed would be someone from VictoriaMetrics, but maybe that was silly of me. :) I don't mind being listed.

Reply:
Added myself.

Reply:
Sorry, I didn't realize I need to add myself there as well. Done.